src/components/common/Buttons/TravelTipsButtons/TravelTipsBtn.jsx

```diff
@@ -0,0 +1,22 @@
+import { memo } from 'react';
+import { cn } from '../../../../utilities/cn';
+
+const TravelTipsBtn = ({ text, className, isActive, onClick }) => {
+  return (
+    <button
+      className={cn(
+        'inline-block h-10 w-auto rounded-10px border px-2 text-center font-fourth text-lg/9 font-normal',
+        isActive
+          ? 'border-dark-color bg-orange-color text-white'
```

Comment:
`border-dark-color` this class is the same for both `isActive === true and === false`. You can move it to static string of classes